servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/Catalogs.java

```diff
@@ -39,11 +39,11 @@ public RESTCatalog getCatalog(Map<String, String> options) {
                   config -> {
                     var builder = HTTPClient.builder(config).uri(config.get(CatalogProperties.URI));
                     config.entrySet().stream()
-                        .filter(e -> e.getKey().startsWith("http.header."))
+                        .filter(e -> e.getKey().startsWith("header."))
```

Comment:
Before Iceberg 1.9.0, the config passed to the `clientBuilder` contained the config-keys with the `http.` prefix, but since 1.9.0, the `http.` prefix is missing. /cc @adutra

Reply:
Yes, it look seem so :shrug: so much for backward compatibility :shrug: